ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
```

Comment:
@hongkailiu are people supposed to start adding these universally now? I know having `cluster` caused problems in the past...

Reply:
This is fine